nav2_route/include/nav2_route/types.hpp

```diff
@@ -194,6 +196,13 @@ struct Node
   }
 };
 
+inline float DirectionalEdge::getEdgeLength()
```

Comment:
Is there a reason this isn't defined in the struct itself?

Reply:
Good point - I think this is the right answer. I don't think a forward declare is enough in this case since we access members of Node.